pkg/remote/remote.go

```diff
@@ -85,11 +85,11 @@ COPY . /okteto/src
 WORKDIR /okteto/src
 
 {{range $key, $val := .OktetoBuildEnvVars }}
-ENV {{$key}} {{$val}}
+ENV {{$key}}={{$val}}
```

Comment:
surrounding the value between `"` is not enough to specify the multi-line value?

Reply:
No, If you surround with "" but don't set the equal everything is in just one line

Reply:
I'm meaning leaving the equal and using the quotes. Do you still need to scape the new line char within `formatEnvVarValueForDocker`?

Reply:
Yes, there is no way of doing it in separate lines in the dockerfile